modules/auxiliary/scanner/http/joomla_api_improper_access_checks.rb

```diff
@@ -0,0 +1,177 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HTTP::Joomla
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Joomla API Improper Access Checks',
+        'Description' => %q{
+          Joomla versions between 4.0.0 and 4.2.7, inclusive, contain an improper API access vulnerability.
+          This vulnerability allows unauthenticated users access to webservice endpoints which contain
+          sensitive information.  Specifically for this module we exploit the users and config/application
```

Comment:
Looks like there's a small bit of extra whitespace here, wasn't sure if that was intentional or not:

<img width="1017" alt="image" src="https://user-images.githubusercontent.com/60357436/233388274-cb011940-06c2-4978-b9c1-2d752b959432.png">

Not a blocker:

```suggestion
          sensitive information. Specifically for this module we exploit the users and config/application
```